bin/zerobufCxx.py

```diff
@@ -962,10 +966,8 @@ def parse_members(self, fbsFile):
                 else:
                     elem_count = int(attrib[4])
                     member = FixedSizeArray(name, value_type, elem_count, self.name)
-                    if is_byte_type:
-                        property['type'] = 'string'
-                        property['media'] = {}
-                        property['media']['binaryEncoding'] = 'base64'
+                    if is_byte_type: # byte arrays are base64 strings
+                        property = _add_base64_string(property)
```

Comment:
@rdumusc is right, property = _add_base64_string(property) doesn't make sense in Python. Functions arguments are not passed by copy, what is copied is the pointer to the object. That means you can't change the variable at the caller scope (but remind the variable is just a pointer), but you can change the object it points to.